sourcebundle/builder_test.go

```diff
@@ -593,6 +686,23 @@ func testingBuilder(t *testing.T, targetDir string, remotePackages map[string]st
 		registryPkgs = append(registryPkgs, pkg)
 	}
 
+	for pkgAddrRaw, deprecations := range registryVersionDeprecations {
+		pkgAddr, err := sourceaddrs.ParseRegistryPackage(pkgAddrRaw)
+		if err != nil {
+			t.Fatalf("invalid registry package address %q: %s", pkgAddrRaw, err)
+		}
+		if registryDeprecations[pkgAddr.Namespace] == nil {
+			registryDeprecations[pkgAddr.Namespace] = make(map[versions.Version]*ModulePackageVersionDeprecation)
+		}
```

Comment:
I don't think you need to reallocate this nested map: https://go.dev/play/p/dmUQYnvF7y_H

Reply:
That's what I thought initially as well, but I'm otherwise getting nil pointer exceptions. Any idea why that could be the case?

Reply:
Actually nevermind that 😅, didn't initially catch that you were assigning a map at the index.

Reply:
Included 👍  https://github.com/hashicorp/go-slug/pull/59/commits/ea8f696aaf4615b5eed02231bed0038ff283c397